cmb2-conditionals.php

```diff
@@ -83,13 +83,10 @@ class CMB2_Conditionals {
 		/**
 		 * Constructor - Set up the actions for the plugin.
 		 */
-		public function __construct() {
-			if ( ! defined( 'CMB2_LOADED' ) || false === CMB2_LOADED ) {
-				return;
-			}
+		public function __construct() {			
 
-			add_action( 'admin_init', array( $this, 'admin_init' ), self::PRIORITY );
-			add_action( 'admin_footer', array( $this, 'admin_footer' ), self::PRIORITY );
+			add_action( 'cmb2_admin_init', array( $this, 'admin_init' ), self::PRIORITY );
+			add_action( 'cmb2_footer_enqueue', array( $this, 'admin_footer' ), self::PRIORITY );
 
```

Comment:
Looks like cmb2_admin_init was added August 30, 2015, during version 2.1.0. cmb2_footer_enqueue was added February 15, 2017 during version 2.2.3.1.  The problem with checking the version of CMB2 before adding these hooks is that often times this plugin could be loaded before CMB is, hence the point of the fork to begin with. I'll clearly annotate the requirement and resubmit.